pkg/gather/service/analyze.go

```diff
@@ -100,6 +101,15 @@ func checkReleaseImageDownload(a analysis) bool {
 	return false
 }
 
+func checkDNSConfig(a analysis) bool {
+	if a.successful {
+		return true
+	}
+	logrus.Error("The DNS config for API and/or API-INT are invalid")
```

Comment:
```suggestion
	logrus.Error("Unable to resolve API and/or API-INT on the bootstrap host.")
```

Is it possible to be more precise? Can we determine whether it is API or API-int that is not resolvable?

Reply:
The error messages from the method that is doing the actual verification does specify which of the URLs are not resolvable. I believe those error messages would be available in the bootstrap log files.
If we want this top level error message to be more specific, we will have to split it into 2 methods, one for each URL.

Reply:
Created 2 different service, one for API URL and the other for API -INT URL which makes the error message specific to the failing URL.